src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  width: $calendar-width;
+  padding-left: $calendar-block-padding;
+  padding-top: $calendar-block-padding * 2; 
+  // Doubled padding-top, 
```

Comment:
Ohhh, man! try  to put the calendar in the middle of the screen. they said about in check list. i can check my solution if you want.

Reply:
I really can`t find where is requirement about calendar centering on display. Could you please write me the number of this requriment in checklist?

Reply:
Thanks! Sometimes my attention is really bad